### PR TITLE
update roadSegment with 2 attributes

### PR DESCRIPTION
**Pull request by gvdwouwer**
I have added status and statusDescription to allow modelling a piece of road that runs over a bridge or lock. When the bridge is raised, no traffic is temporarily possible on the roadsegment. The attributes could also be used to model other types of events (such as roadworks) that temporarily limit or prohibit traffic.

The file passed the check_scheme validator in the learning zone.

### CODE REVIEW

**Review by albertoabellagarcia — CHANGES_REQUESTED** [review]
Thank you for the contribution
1) I need you to extend the example with the values for the new attributes. files in the examples' directory
2) Needs the word 'Property. ' in the descriptions of the attributes according to the [contribution manual](https:/bit.ly/contribution_manual)
3) It is also required to accept the [contribution agreement](https://smartdatamodels.org/index.php/contributor-agreement/)
4) if you want to appear listed in the [contributors](https://github.com/smart-data-models/dataModel.Transportation/blob/master/CONTRIBUTORS.yaml) modify the PR accordingly
5) Please also extend the [adopters.yaml](https://github.com/smart-data-models/dataModel.Transportation/blob/master/RoadSegment/ADOPTERS.yaml) file with the use case

**Comment by gvdwouwer** [comment]
I have made these changes and pushed them to github: 
* added Property to the description in schema.json
* adopters.yaml updated
* accepted the contribution agreement
* The examples are updated. However, for the LD examples, I think the normalised and the key-value example has been switched in the original repo. Please advise if that should be corrected.

I would like to be listed as a contributer as you mentioned in your 4th item. But I do not know how I should modify the PR.

File: RoadSegment/examples/example.jsonld [hunk 1]
```diff
@@ -1,101 +1,109 @@
 {
-    "id": "urn:ngsi-ld:RoadSegment:Spain-RoadSegment-A62-osm-24702186",
```
Comment (albertoabellagarcia) [inline RoadSegment/examples/example.jsonld:2]: it should be example-normalized.jsonld

**Review by albertoabellagarcia — COMMENTED** [review]
Attend the minor issues and ready to be published. don't you want to appear at the contributors.yaml file?

File: RoadSegment/examples/example-geojsonfeature.json [hunk 2]
```diff
@@ -1,128 +1,136 @@
 {
-  "id": "urn:ngsi-ld:RoadSegment:Spain-RoadSegment-A62-osm-24702186",
```
[the remark this answers is not in the document]
Comment (gvdwouwer) [reply RoadSegment/examples/example-geojsonfeature.json]: reverted example-geojsonfeature.json to the original file

File: RoadSegment/examples/example-normalized.jsonld [hunk 3]
```diff
@@ -1,64 +1,66 @@
 {
-    "id": "urn:ngsi-ld:RoadSegment:Spain-RoadSegment-A62-osm-24702186",
```
[the remark this answers is not in the document]
Comment (gvdwouwer) [reply RoadSegment/examples/example-normalized.jsonld:2]: done

File: RoadSegment/schema.json [hunk 4]
```diff
@@ -1,157 +1,173 @@
...
-    "id",
-    "name",
-    "type",
-    "refRoad",
-    "startPoint",
-    "endPoint",
-    "allowedVehicleType"
-  ]
+	"$schema": "http://json-schema.org/schema#",
+	"$schemaVersion": "0.2.0",
```
[the remark this answers is not in the document]
Comment (gvdwouwer) [reply RoadSegment/schema.json]: done

**Comment by albertoabellagarcia** [comment]
Ready to be published

**Pull request merged**